books/free-programming-books-pt_BR.md

```diff
@@ -288,7 +287,6 @@
 
 * [Adicionar Bootstrap e Font-awesome no projeto criado com o Vue Cli](https://web.archive.org/web/20180613054310/http://www.vedovelli.com.br/frontend/adicionar-bootstrap-e-font-awesome-no-projeto-criado-com-o-vue-cli/)
 * [Documentação pt-BR](https://br.vuejs.org/v2/guide/)
-* [Vue.js Brasil - Artigos em Português sobre Vue.js](http://www.vuejs-brasil.com.br)
```

Comment:
A replacement could be... https://vuejsbrasil.org/

Reply:
but it's a blog

Reply:
Let's Eric decide

Reply:
it's a blog

Reply:
so it's ready to be merged I guess